takwimu/takwimu_ui/src/components/FeaturedAnalysis/index.js

```diff
@@ -34,32 +34,48 @@ class FeaturedAnalysis extends React.Component {
   render() {
     const {
       classes,
-      takwimu: { featured_analysis: featuredAnalysis }
+      takwimu: {
+        page: {
+          featured_analysis: { value: featuredAnalysis }
+        }
+      }
     } = this.props;
+    if (!featuredAnalysis) {
+      return null;
+    }
+
+    const {
+      title,
+      featured_analyses: featuredAnalyses,
```

Comment:
typo `featured_analyses` -> `featured_analysis`

Reply:
Not a typo, plural form: https://writingexplained.org/analysis-vs-analyses-plural